marklogic-data-hub-central/build.gradle

```diff
@@ -179,24 +178,13 @@ task copyUiFiles(type: Copy, dependsOn: deleteSpringBootUiFiles, group: taskGrou
     into springBootUiPath
 }
 
-// Before the Spring Boot jar is built, build the UI files and copy them over
-if (!(
-    gradle.startParameter.taskNames*.toLowerCase().contains("publishplugins") ||
-        gradle.startParameter.taskNames*.toLowerCase().contains("bintrayUpload") ||
-        gradle.startParameter.taskNames*.toLowerCase().contains("javadoc") ||
-        // bootRun doesn't need the UI, as it's assumed that the UI is being run separately via npm start
-        gradle.startParameter.taskNames*.toLowerCase().contains("bootrun") ||
-        gradle.startParameter.taskNames*.toLowerCase().contains("generateschemapojos") ||
-        project.hasProperty('skipui')
-)
-) {
+// When creating the HC war file, need to build the UI files and add them to the classpath so that the war file includes them
+if (gradle.startParameter.taskNames*.toLowerCase().contains("bootwar")) {
     tasks.processResources.dependsOn(buildUi, copyUiFiles)
 }
 
+
 test {
-	exclude 'com/marklogic/hub/curation/integrationtests/**'
```

Comment:
Are we going to run all these tests on PR too?

Reply:
None of the 3 excluded packages exist anymore